daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -669,9 +670,70 @@ func (devices *DeviceSet) loadMetadata(hash string) *DevInfo {
 	return info
 }
 
+func getDeviceUUID(device string) (string, error) {
+	out, err := exec.Command("blkid", "-s", "UUID", "-o", "value", device).Output()
```

Comment:
there is a header and way to link to libbklid, but that seems too much. Please would require additional build time packages, where `blkid` is provided by `util-linux` which is on every major distro (even debian minbase).